test/powershell/Modules/Microsoft.PowerShell.Utility/Environment-Variables.Tests.ps1

```diff
@@ -40,4 +40,10 @@ Describe "Environment-Variables" -Tags "CI" {
 	$ENV:TESTENVIRONMENTVARIABLE | Should Be $expected
 
     }
+
+    It "User Agent should reflect current plafformID" {
```

Comment:
This test should be in `test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1`